bindings/cs/rl.net.native/rl.net.live_model.cc

```diff
@@ -162,6 +162,16 @@ API int LiveModelReportOutcomeJson(livemodel_context_t* context, const char* eve
   return context->livemodel->report_outcome(event_id, outcomeJson, status);
 }
 
+API int LiveModelReportOutcomeSlotF(livemodel_context_t* context, const char* event_id, int slotIndex, float outcome, reinforcement_learning::api_status* status)
```

Comment:
nit
```suggestion
API int LiveModelReportOutcomeSlotF(livemodel_context_t* context, const char* event_id, int slot_index, float outcome, reinforcement_learning::api_status* status)
```

Reply:
Makes sense. I changed outcomeJson to outcome_json for unity.